CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.2.5
+
+- [Android] Fix a uncommon issue with this.getCurrentVersion returning null inside of showInAppMessageIfAvailable
```

Comment:
Minor point but I believe `this.getCurrentVersion` should be `this.getCurrentActivity`